P4Plugin/Source/P4LoginCommand.cpp

```diff
@@ -19,6 +19,7 @@ class P4LoginCommand : public P4Command
 
 		m_LoggedIn = false;
 		m_Password = task.GetP4Password();
+		//m_CheckingForLoggedIn = args.size() > 1;
```

Comment:
Can this be removed? What's the consequence for not setting `m_CheckingForLoggedIn`?

Reply:
This is a waste from the rebase from master we did. Removing theme